test/e2e/bin_packing/bin_packing_test.go

```diff
@@ -195,8 +195,8 @@ var _ = Describe("", func() {
 	AfterEach(func() {
 		testDescription := ginkgo.CurrentSpecReport()
 		if testDescription.Failed() {
-			tests.LogTestClusterInfoWrapper(testDescription.FailureMessage(), []string{ns})
-			tests.LogYunikornContainer(testDescription.FailureMessage())
+			tests.LogTestClusterInfoWrapper(testDescription.FullText(), testDescription.FailureMessage(), []string{ns})
+			tests.LogYunikornContainer(testDescription.FullText())
```

Comment:
Note: This was a bug. Fix it in this PR. 
(Was: Pass in testDescription.FailureMessage() as testName)